media-libs/libfishsound/libfishsound-1.0.0-r1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Simple programming interface for decoding and encoding audio data using vorbis or speex"
+HOMEPAGE="https://www.xiph.org/fishsound/"
+SRC_URI="https://downloads.xiph.org/releases/libfishsound/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="flac speex"
+
+RDEPEND="media-libs/libvorbis
+	media-libs/libogg
+	flac? ( media-libs/flac )
+	speex? ( media-libs/speex )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+# bug #395153
+RESTRICT="test"
+
+PATCHES=( "${FILESDIR}"/${P}-pc.patch )
+
+src_prepare() {
+	sed -i \
+		-e 's:doxygen:doxygen-dummy:' \
+		configure || die
+
+	default
+}
+
+src_configure() {
+	econf \
+		--disable-dependency-tracking \
+		$(use_enable flac) \
+		$(use_enable speex)
```

Comment:
are you sure the configure script is not buggy and --enable-speex/flac does not actually disable it ?

Reply:
You're right. The configure script is buggy. I've reverted  back to the previous use.